src/core/sys/posix/utime.d

```diff
@@ -35,53 +35,10 @@ struct utimbuf
 int utime(in char*, in utimbuf*);
 */
 
-version( linux )
-{
-    struct utimbuf
-    {
-        time_t  actime;
-        time_t  modtime;
-    }
-
-    int utime(in char*, in utimbuf*);
-}
-else version( OSX )
-{
-    struct utimbuf
-    {
-        time_t  actime;
-        time_t  modtime;
-    }
-
-    int utime(in char*, in utimbuf*);
-}
-else version( FreeBSD )
-{
-    struct utimbuf
-    {
-        time_t  actime;
-        time_t  modtime;
-    }
-
-    int utime(in char*, in utimbuf*);
-}
-else version( Solaris )
+struct utimbuf
 {
-    struct utimbuf
-    {
-        time_t  actime;
-        time_t  modtime;
-    }
-
-    int utime(in char*, in utimbuf*);
+    time_t  actime;
+    time_t  modtime;
 }
-else version( Android )
-{
-    struct utimbuf
-    {
-        time_t  actime;
-        time_t  modtime;
-    }
 
-    int utime(in char*, in utimbuf*);
-}
+int utime(in char*, in utimbuf*);
```

Comment:
[Defined this way in the POSIX standard](http://pubs.opengroup.org/onlinepubs/9699919799/functions/utime.html).